articles/client-platforms/react/05-linking-accounts.md

```diff
@@ -0,0 +1,372 @@
+      'Content-Type': 'application/json',
+      'Authorization': 'Bearer ' + this.getToken()
+    }
+
+    const userId = this.getProfile().user_id
+    return fetch(`<%= "https://${this.domain}/api/v2/users/${userId}/${url}" %>`, {
+      headers,
+      ...options
+    })
+    .then(response => response.json())
+  }
+
+  linkAccount(token){
+    // prepares api request body data
+    const data = {
+      link_with: token
+    }
+    // sends a post to auth0 api to create a new identity
+    return this.fetchApi('identities', {
+      method: 'POST',
+      body: JSON.stringify(data)
+    })
+    .then(identities => {
+      const profile = this.getProfile()
+      this.setProfile({...profile, identities}) // updates profile identities
+    })
+  }
+}
+```
+
+The code introduces also two new methods: `fetchApi` to send requests to auth0 users api with the required headers and parsing the requests to json. Finally, `linkAccount` uses `fetchApi` to send a _POST_ request and create a new identity in user account, updating the stored profile after a successful response. For more details, check the [user identities endpoint documentation](https://auth0.com/docs/api/management/v2#!/Users/post_identities).
+
+Now you're able to update the `LinkedAccountsList` component to render a __Link Account__ button using the `LinkAccountService.link` method:
+
+```javascript
+/* ===== ./src/components/LinkedAccount/LinkedAccountsList.js ===== */
+import React, { PropTypes as T } from 'react'
+import LinkAccountService from 'utils/LinkAccountService'
+...
+
+export class LinkedAccountsList extends React.Component {
+  ... //omitting some code
+
+  render(){
+    const { profile, auth } = this.props
+    const linker = new LinkAccountService(auth) // initializing the new helper
+    let items = []
+    if (profile) {
+      items = profile.identities.map(identity => {
+        return (<LinkedAccountItem {...this.props} identity={identity} />)
+      })
+    }
+
+    return (
+      <div className={styles.root}>
+        <h3>Linked Accounts</h3>
+        <ListGroup>{items}</ListGroup>
+        // the button to call auth0lock and show sign in window
+        <Button onClick={linker.link} bsStyle="primary">Link Account</Button>
+      </div>
+    )
+  }
+}
+
+export default LinkedAccountsList;
+```
+
+Running the application, you'are now able to click on __Link Account__ button in the user home page and link a _facebook_ or _google_ account to it. After the successful link, the new identity is showed in the __Linked Accounts__ list.
+
+## 3. Un-Linking Accounts
+
+You can also dissociate a linked account just hitting the [unlink user account endpoint](https://auth0.com/docs/api/management/v2#!/Users/delete_provider_by_user_id) in auth0 api. You need to send the primary user_id, and the provider/user_id of the identity you want to unlink. Let's update `AuthService` to, besides a `LinkAccount` method, provide also an `UnlinkAccount` method:
```

Comment:
@flaviogranero 
Update the link to `/api/management/v2#!/Users/delete_provider_by_user_id` and fix `dissociate ` typo.

Maybe the  "user_id" and "provider/user_id" should be like that? -> `user_id` and `provider/user_id`